python/exercise1/vowel_counter.py

```diff
@@ -2,17 +2,19 @@
 
 def num_vowels(text):
```

Comment:
Maybe add a docstring to describe what your functions do to make it clearer to the people using the tool.